src/main/java/OopEx2.java

```diff
@@ -0,0 +1,22 @@
+import java.util.Stack;
+public class OopEx2 {
+    //Push bổ sung một phan tử vào dinh (top) của ngan xếp, nghia là sau các
+    //phan tử đã có trong ngan xếp. Pop giai phong và tra ve phan tu dang
+    //dung o dinh cua ngan xep.
+    public void stack_push(Stack<Integer> stack, int i) {
```

Comment:
Đề bài là implement Stack mà em đi lấy lại nó thì nói làm gì

```suggestion
    public void stack_push(int i) {
```

Reply:
Done